frontend/components/shelter/home/NoData.tsx

```diff
@@ -1,17 +1,6 @@
 import { Button } from "antd";
 import styled from "styled-components";
-
-const CenteredDiv = styled.div`
-	display: flex;
-	justify-content: center;
-`;
-
-const Container = styled.div`
-	height: calc(100vh - var(--menu-header-menu-height));
-	display: flex;
-	flex-direction: column;
-	justify-content: center;
-`;
+import { Container } from "./PetsTable/EmptyComtainer";
```

Comment:
please fix spelling for "EmptyComtainer"

Reply:
updated